api/python/provisioner/commands/set_swupgrade_repo.py

```diff
@@ -145,6 +146,47 @@ def _single_repo_validation(self, release, repo_name):
                     )
                 )
 
+    @staticmethod
+    def _get_hash_params(
+            params: inputs.SWUpgradeRepo) -> Tuple[bytes, str, str]:
+        """
+        Parse and validate the provided hash parameters
+
+        Parameters
+        ----------
+        params: Type[inputs.SWUpgradeRepo]
+            input parameters which contain the hash parameters
+
+        Returns
+        -------
+        Tuple
+            Returns tuple with hash_sum, hash_type and file_name
+
+        """
+        hash_sum = None
+        hash_type = None
+        file_name = None
+
+        if Path(params.hash_str).exists():
+            _data = load_checksum_from_file(Path(params.hash_str))
+            hash_sum, hash_type, file_name = _data
```

Comment:
From my point of view it is not necessary because it is pythonic way to return many arguments from function.  But if you think it will make code the code more clean, I don't mind. Fixed that.

Reply:
Not sure it is pythonic one. Do you yourself think is convenient to deal with these three tuples each time? What if you decide later to make it a tuple of four?